services/core/shared/core/compat/sdk_v5/schema/votesAggregate.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+module.exports = {
+	primaryKey: 'id',
+	schema: {
+		id: { type: 'string' },
+		sentAddress: { type: 'string' },
+		receivedAddress: { type: 'string' },
+		amount: { type: 'bigInteger', defaultValue: 0 },
+		timestamp: { type: 'integer' },
```

Comment:
What timestamp is presented here? If it's last update it has to be mentioned in the docs.

Reply:
Yes, its last update time (last vote received to given account from any account). It's only for internal purpose and we dont send it in response. Also, I have update the docs to include `aggregate` params for `votes_received` now.